ui/postcss2-index.js

```diff
@@ -0,0 +1,309 @@
+  ) {
+    for (let key of __getOwnPropNames(module2))
+      if (!__hasOwnProp.call(target, key) && key !== 'default')
+        __defProp(target, key, {
+          get: () => module2[key],
+          enumerable:
+            !(desc = __getOwnPropDesc(module2, key)) || desc.enumerable,
+        })
+  }
+  return target
+}
+var __toModule = (module2) => {
+  return __exportStar(
+    __markAsModule(
+      __defProp(
+        module2 != null ? __create(__getProtoOf(module2)) : {},
+        'default',
+        module2 && module2.__esModule && 'default' in module2
+          ? { get: () => module2.default, enumerable: true }
+          : { value: module2, enumerable: true }
+      )
+    ),
+    module2
+  )
+}
+__markAsModule(exports)
+__export(exports, {
+  default: () => src_default,
+})
+var import_fs_extra = __toModule(require('fs-extra'))
+var import_util = __toModule(require('util'))
+var import_path = __toModule(require('path'))
+var import_tmp = __toModule(require('tmp'))
+var import_postcss2 = __toModule(require('postcss'))
+var import_postcss_modules = __toModule(require('postcss-modules'))
+var import_less = __toModule(require('less'))
+var import_stylus = __toModule(require('stylus'))
+var import_resolve_file = __toModule(require('resolve-file'))
+const postCSSPlugin = ({
+  plugins = [],
+  modules = true,
+  rootDir = process.cwd(),
+  sassOptions = {},
+  lessOptions = {},
+  stylusOptions = {},
+}) => ({
+  name: 'postcss2',
+  setup(build) {
+    let cache = new Map()
+    // cache = {
+    //   'srcPath': {
+    //     lastMtimeMs: 1634030364414,
+    //     output: ''
+    //   }
+    // }
+
+    const tmpDirPath = import_tmp.default.dirSync().name,
+      modulesMap = []
+    const modulesPlugin = (0, import_postcss_modules.default)({
+      generateScopedName: '[name]__[local]___[hash:base64:5]',
+      ...(typeof modules !== 'boolean' ? modules : {}),
+      getJSON(filepath, json, outpath) {
+        const mapIndex = modulesMap.findIndex((m) => m.path === filepath)
+        if (mapIndex !== -1) {
+          modulesMap[mapIndex].map = json
+        } else {
+          modulesMap.push({
+            path: filepath,
+            map: json,
+          })
+        }
+        if (
+          typeof modules !== 'boolean' &&
+          typeof modules.getJSON === 'function'
+        )
+          return modules.getJSON(filepath, json, outpath)
+      },
+    })
+
+    build.onResolve(
+      { filter: /.\.(css|sass|scss|less|styl)$/ },
+      async (args) => {
+        const start = Date.now()
+
+        if (args.namespace !== 'file' && args.namespace !== '') return
+
+        let sourceFullPath = (0, import_resolve_file.default)(args.path)
+        if (!sourceFullPath)
+          sourceFullPath = import_path.default.resolve(
+            args.resolveDir,
+            args.path
+          )
+        // hack
+        let exist = await import_fs_extra.exists(sourceFullPath + '.js')
+        if (exist) {
+          return
+        }
```

Comment:
To fix the errors that it reports can't find `.scss` files when compiling fluent-ui, because those `.scss` files are `.scss.js` files. We need to report the issue to the upstream later.